partner_contact_birthplace/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Simone Rubino - Agile Business Group
+# Copyright 2019 Nicolò Crisafulli - Agile Business Group
```

Comment:
@Crisa221 Removed  `Copyright` Line, Remove in Other Place Also
You can only Migrate this module so you can be adding a name in the `contribution` section

Reply:
@nikul-serpentcs Do I have to remove only my line or also the line I was carrying from version 10?

Reply:
Done